app/services/publisher_notifier_unverified.rb

```diff
@@ -0,0 +1,76 @@
+class PublisherNotifierUnverified < BaseService
+  attr_reader :notification_params, :notification_type
+
+  PUBLISHER_TYPES = %w(domain) # TODO: youtube
+
+  def initialize(publisher_type:, publisher_id: )
+    @publisher_id = publisher_id
+    @publisher_type = publisher_type
+
+    ensure_params_exist
+
+    if !PUBLISHER_TYPES.include?(@publisher_type)
+      raise InvalidPublisherTypeError.new("#{@publisher_type} is an invalid publisher type")
+    end
+  end
+
+  def perform
+    if @publisher_type == "domain"
```

Comment:
minor: we plan on adding more platforms in the future, so a `case` statement might be more sense. however, we can do that later.